lang/cs/Org.Apache.REEF.Driver/Bridge/Events/RunningTask.cs

```diff
@@ -60,12 +60,12 @@ public void Send(byte[] message)
 
         public void Suspend(byte[] message)
         {
-            throw new NotImplementedException();
+            _runningTaskClr2Java.Suspend(message);
         }
 
         public void Suspend()
```

Comment:
What's the reasoning behind the need for consolidation? One sends a message and one doesn't.